utkdigitalinitiatives/polk-correspondence-app
# Address PTC-134 or The One for Those Who Live in the Woods.

**JIRA Ticket**: [PTC-134](https://jirautk.atlassian.net/projects/PTC/issues/PTC-134)

# What does this Pull Request do?

Nullifies large aspects of the React header functionality for the short term while keeping intact the general layout.

# What's new?

* Hides the Menu and Search buttons in the header.
* Adds `<noscript>...</noscript>` as a fallback for users without javascript.
* Makes menu items in super header real.

# How should this be tested?

1. `ant` and upload.
2. View any Polk app page in browser, menu should display similar to previous with Menu and Search buttons hidden. A rendered menu in the light gray super header at the top of the page will have working links. Click each one and make sure they land somewhere.
3. Open a new browser tab and disable Javascript. This is easiest from safari under Preferences > Advanced. Repeat step two.

# Additional Notes:
The is packages together fixes for a generic header while also addressing a general accessibiltiy concern for users who choose to use ancien regime browsers OR to disable JS for whatever silly reason. The `<noscript>...` renders flat HTML that retains only the most rudimentary functionality.

# Interested parties
@saltar
